### ISSUE TITLE
[docs] provide more docs

### ISSUE THREAD

**Issue by celrenheit** [celrenheit/sandglass#9]
Although the project is moving quickly, documentation is a necessity and will be prioritized.

**Comment by stern0** [comment]
I agree, the current documentation is quite good but there should be more documentation on the service usage for example with the Homebrew installation, and also maybe a guide for developing clients.
